src/courses/models.py

```diff
@@ -248,6 +254,9 @@ def num_days(self, upto_today=False):
         # We want to return an int because a numpty.int64 is not JSON serializable as of the moment
         return count.item() if hasattr(count, 'item') else count
 
+    def get_name(self):
```

Comment:
What the purpose of a getter for name?  Just access it directly.

Reply:
removed